### PR TITLE
Extravaganza blog post for January 2016.

### PR DESCRIPTION
**Pull request by Osmose**
@jgmize @erikrose @wfwalker @pmclanahan @jpetto @darkwing r?

I'll be posting this Monday morning, so please leave any feedback or request any changes before then. Thanks!

### CODE REVIEW

**Comment by jgmize** [comment]
lgtm

File: blog/extravaganza/2016-01-January.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+[Pmac]: https://mozillians.org/en-US/u/pmac/
+[django-mozilla-product-details]: https://github.com/mozilla/django-product-details
+
+## New Hires / Interns / Volunteers / Contributors
+Here we introduce any newcomers to the Webdev group, including new employees,
+interns, volunteers, or any other form of contributor.
+
+- [jpetto][] is switching from a contractor to a full-time web developer on the
+  Engagement Web Development team!
+- [davidwalsh][] is moving to the Web App Developer Initiative team!
```
Comment (darkwing) [inline blog/extravaganza/2016-01-January.md]: Works for me!  Thanks for including me!

File: blog/extravaganza/2016-01-January.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+[Pmac]: https://mozillians.org/en-US/u/pmac/
+[django-mozilla-product-details]: https://github.com/mozilla/django-product-details
+
+## New Hires / Interns / Volunteers / Contributors
+Here we introduce any newcomers to the Webdev group, including new employees,
+interns, volunteers, or any other form of contributor.
+
+- [jpetto][] is switching from a contractor to a full-time web developer on the
+  Engagement Web Development team!
```
Comment (jpetto) [inline blog/extravaganza/2016-01-January.md]: :smile:

File: blog/extravaganza/2016-01-January.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+[Pontoon team]: mailto:pontoon@mozilla.org
+
+### Mozilla.org Dual SHA1/SHA256 Certificate Negotiation
+Next was [jgmize][] with info about [www.mozilla.org][]'s recent work around
+enabling both SHA1 and SHA256 certificates to be used on the site. Firefox is
+[phasing out support for SHA1 certificates][] along with the other major
+browsers, but users on older browsers need to be able to download new versions
+of Firefox from www.mozilla.org. Some of these older versions are from before
+browsers supported SHA256 certificates. In order to avoid leaving these users
+on out-dated and insecure Firefox versions, www.mozilla.org needs to be able to
```
Comment (pmclanahan) [inline blog/extravaganza/2016-01-January.md]: The problem is not old Fx versions. Fx has its own SSL stack and has supported SHA2 for years. The problem is people without a modern browser on an OS like Win XP SP2. Maybe reword with something like:

> In order to avoid leaving these users stuck without a way to get a modern browser, www.mozilla.org needs to be able to fall back to SHA1 certificates when necessary.

**Comment by pmclanahan** [comment]
r+

I suggested a wording change, but it's not that big a deal. Take or leave it as you wish. Thanks for writing these up! :cake:

File: blog/extravaganza/2016-01-January.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+[Node.js]: https://nodejs.org/en/
+[ServiceWorker Cookbook]: https://serviceworke.rs/
+[ServiceWorkers]: https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API
+
+### DXR Updates
+ErikRose shared a slew of [DXR][] updates:
+
+- Recognizing overrides of virtual methods
+- Recognizing multiple directly overridden virtual methods
+- Indexing of [Cargo][] packages
```
Comment (erikrose) [inline blog/extravaganza/2016-01-January.md]: Let's say "The ability to index Cargo packages", since actually indexing All The Cargo Packages is a separate thing we've been talking about.

File: blog/extravaganza/2016-01-January.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+## Roundtable
+The Roundtable is the home for discussions that don't fit anywhere else.
+
+### Docker Practices for Development
+ErikRose closed us out with a few tips for using [Docker][] for developing a
+website:
+
+- [Store your virtualenv on a data volume](https://github.com/mozilla/dxr/blob/master/tooling/docker/docker-compose.yml#L19-L22)
+- [Create virtualenvs using a Makefile](https://github.com/mozilla/dxr/blob/master/makefile#L76-L81)
+- [Add a Makefile command for interactive debugging](https://github.com/mozilla/dxr/blob/master/makefile#L44-L50)
```
Comment (erikrose) [inline blog/extravaganza/2016-01-January.md]: Makefile *target* is the proper vocab

**Comment by erikrose** [comment]
Thanks for writing these up!
